app/controllers/idv/sessions_controller.rb

```diff
@@ -5,12 +5,20 @@ class SessionsController < ApplicationController
     before_action :confirm_two_factor_authenticated
 
     def destroy
-      analytics.track_event(Analytics::IDV_VERIFICATION_ATTEMPT_CANCELLED)
-      Idv::CancelVerificationAttempt.new(user: current_user).call
+      cancel_verification_attempt_if_pending_profile
+      analytics.track_event(Analytics::IDV_START_OVER)
```

Comment:
@aduth Is it possible that there could be a log event for both start over and cancel? It seems like it might be possible but I haven't dug deeply into how `cancel_verification_attempt_if_pending_profile` and `destroy` play together.

Reply:
> @aduth Is it possible that there could be a log event for both start over and cancel? It seems like it might be possible but I haven't dug deeply into how `cancel_verification_attempt_if_pending_profile` and `destroy` play together.

If I understand correctly, yes it's possible and desirable, at least in the scenario handled already by this controller where "cancel" is actually the GPO verification "Clear my information and start over" link, which would be counted both as a sort of cancellation (existed previously) and as starting over.

<a href="https://user-images.githubusercontent.com/1779930/133323036-a8e146c0-ea19-4b67-905f-b1095e6fa319.png"><img src="https://user-images.githubusercontent.com/1779930/133323036-a8e146c0-ea19-4b67-905f-b1095e6fa319.png" alt="clear and start over screenshot" width="450"></a>

The changes in this pull request essentially piggy-back off this GPO "clear and start over" behavior for _all_ "Start Over", but only log `IDV_VERIFICATION_ATTEMPT_CANCELLED` in situations where it would have been logged previously.

Reply:
Herm, ok. It seems weird to me to log both a start over and cancel event for what I think is a single user event if I'm understanding correctly. It still seems like it's either a start over or cancel not both.

Reply:
> Herm, ok. It seems weird to me to log both a start over and cancel event for what I think is a single user event if I'm understanding correctly. It still seems like it's either a start over or cancel not both.

We could make it an either / or ? i.e. "Clear my information and start over" continues to log only `IDV_VERIFICATION_ATTEMPT_CANCELLED`, or otherwise logging the `IDV_START_OVER` event.

Alternatively, I could change this not to try to generalize the GPO behavior so much and create something new instead, with a similar session reset behavior.

Reply:
Follow-up at https://github.com/18F/identity-idp/pull/5394#issuecomment-920157121.